core/src/main/scala/org/apache/spark/h2o/backends/external/ProxyStarter.scala

```diff
@@ -30,9 +30,9 @@ import org.spark_project.jetty.util.thread.{QueuedThreadPool, ScheduledExecutorS
 
 object ProxyStarter extends Logging {
   def startFlowProxy(conf: H2OConf): URI = {
+    var port = findNextFreeFlowPort(conf.clientWebPort, conf.clientBasePort)
```

Comment:
if the first try to get a free port fails, it will cause failure of the whole process. Is it what we want?

Reply:
Good catch, nope, thanks!